src/index.html

```diff
@@ -1,12 +1,478 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__content">
+    <header class="header-menu page__header-menu">
+      <a class="logo header-menu__logo" href="./index.html"> 
+        <img
+          src="./images/logo.png"
+          alt="Mate academy school logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__marker">
+            <a
+              href="#Apple"
+              class="nav__link nav__link--active is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__marker">
+            <a
+              href="#Samsung"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__marker">
+            <a
+              href="#Smartphones"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__marker">
+            <a
+              href="#Laptops & Computers" 
+              class="nav__link"
+              data-qa="nav-hover" 
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__marker">
+            <a
+              href="#Gadgets"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__marker">
+            <a
+            href="#Gadgets"
+            class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__marker">
+            <a
+              href="#Gadgets"
```

Comment:
The href attribute should have a unique value. Change the href to match the correct destination